### PR TITLE
List: Icon Left/Right (Wearable): add example and style

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
Issue]
      https://github.com/Samsung/TAU/issues/1056,
      https://github.com/Samsung/TAU/issues/1057
    
[Problem] Lack of example and out dated style
[Solution] Add example and update style
    
Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
I will join Icon Right implementation since it's similar to Icon Left.

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less [hunk 1]
```diff
@@ -85,14 +85,45 @@
...
 				&-right {
-					padding-right: 100 * @unit_base;
+					& > * {
+						right:  78 * @px_base;
+						margin-left: 92 * @px_base; // to set width element 222px
+						height: 50 * @px_base;
+					}
+
+					.ui-li-thumb-right {
+						right: 16px;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/wearable/changeable/theme-circle/arclistview.less]: px -> `@px_base`

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less [hunk 2]
```diff
@@ -85,14 +85,45 @@
...
 				&-left {
-					padding-left: 100 * @unit_base;
+					& > * {
+						left:  78 * @px_base;
+						margin-right: 92 * @px_base; // to set width element 222px
+						height: 50 * @px_base;
+					}
+
+					.ui-li-thumb-left {
+						left: 16px;
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/arclistview.less]: please use @px_base.

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less [hunk 3]
```diff
@@ -85,14 +85,45 @@
 		li {
 			/* below style is not prefixed "ui-" for backward compatibility */
 			&.li-has-thumb {
+
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/arclistview.less]: delete 1 line.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		UI Components
+	</title>
+	<link href="../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../css/style.circle.css"  rel="stylesheet" />
+	<script src="../../../../js/circle-helper.js">
+	</script>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html]: In my personal opinion, it's nice to see that the last tag is moved to shorten the line.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	<script src="../../../../lib/tau/wearable/js/tau.js">
+	</script>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html]: ditto.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		UI Components
+	</title>
+	<link href="../../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<link href="../../../../css/style.circle.css"  rel="stylesheet" />
+	<script src="../../../../js/circle-helper.js">
+	</script>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html]: In my personal opinion, it's nice to see that the last tag is moved to shorten the line.

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+	<script src="../../../../lib/tau/wearable/js/tau.js">
+	</script>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html]: ditto.

**Review by DohyungLim — COMMENTED** [review]
6 comment.

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less (hunk 2 above)
Comment (TomaszLukawskiSam) [inline src/css/profile/wearable/changeable/theme-circle/arclistview.less]: px -> `@px_base`

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]
Units need fix

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/arclistview.less]: Done

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less (hunk 2 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/arclistview.less]: Done

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less (hunk 3 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/arclistview.less]: Done

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html (hunk 4 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html]: Done

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html (hunk 5 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/icon-left.html]: Done

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html (hunk 6 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html]: Done

File: examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html (hunk 7 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/main/icon-right.html]: Done

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less (hunk 2 above)
Comment (grzegorz-czajkowski) [reply src/css/profile/wearable/changeable/theme-circle/arclistview.less]: Done

**Review by TomaszLukawskiSam — APPROVED** [review]

**Pull request merged**